internal/commands/pull/command.go

```diff
@@ -60,32 +61,58 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		return err
 	}
 
-	pathTarget, zipPkg, err := cmd.doExport(profile, clients.Realm, app.GroupID, app.ID)
+	clientZipPkgs, err := cmd.inputs.resolveClientTemplates(ui, clients.Realm, app.GroupID, app.ID)
 	if err != nil {
 		return err
 	}
 
-	pathRelative, err := filepath.Rel(profile.WorkingDirectory, pathTarget)
+	projectPath, zipPkg, err := cmd.doExport(profile, clients.Realm, app.GroupID, app.ID)
 	if err != nil {
 		return err
 	}
 
-	proceed, err := checkAppDestination(ui, pathTarget)
+	var pathFrontend string
+	pathBackend := projectPath
+	if len(clientZipPkgs) != 0 {
+		pathFrontend = filepath.Join(projectPath, local.FrontendPath)
+		if proceed, err := checkPathDestination(ui, pathFrontend); err != nil {
+			return err
+		} else if !proceed {
+			return nil
+		}
+		pathBackend = filepath.Join(projectPath, local.BackendPath)
+	}
+
+	// App path
+	proceed, err := checkPathDestination(ui, pathBackend)
```

Comment:
let's use `pathProject` instead of `pathBackend` here.  this gives use one check/prompt to the user prior to the potential dry-run exit.  if the `pathProject` exists, chances are the `./backend` and `./frontend` paths already exist as well (unless they somehow switched their app from not using templates to using templates)